doc/orchestrator.rst

```diff
@@ -165,3 +165,313 @@ A full example of configuring KeyDB to run in SmartSim is shown below
   export REDIS_CONF=/path/to/keydb.conf
 
   # run smartsim workload
+
+Multiple Orchestrators
+======================
+
+SmartSim offers functionality to automate the deployment of multiple
+databases on an HPC cluster, supporting workloads that require multiple
+``Orchestrators`` for an ``Experiment``. For instance, a workload may consist of a
+simulation with high inference performance demands, along with an analysis and
+visualization workflow connected to the simulation. In such cases, SmartSim
+allows you to augment the number of clustered orchestrators or provision multiple
+co-located orchestrators, ensuring that the workload can scale to meet its resource
+requirements. Below is a general example demonstrating the process of launching
+multiple databases with unique identifiers and distributing data between them.
+
+Defining workflow stages requires the utilization of functions associated
+with the ``Experiment`` object. The Experiment object is intended to be instantiated
+once and utilized throughout the workflow runtime. Begin by creating a ``Experiment``
+object and assign it to the ``exp`` variable. In this example, we instantiate the
+``Experiment`` object with the name ``getting-started-multidb``.
+
+.. code-block:: python
+
+  import numpy as np
+  from smartredis import Client
+  from smartsim import Experiment
+
+  exp = Experiment("getting-started-multidb", launcher="auto")
+
+In the context of this ``Experiment``, it's essential to create and launch
+the databases as a preliminary step before any other components. We aim
+to showcase the multi-database automation capabilities of SmartSim, so we
+create two types of Orchestrators in the workflow: a co-located Orchestrator and a
+clustered Orchestrator.
+
+.. note::
+
+  An Orchestrator object supports the deployment of the database. 
+
+To create a Orchestrator, you can utilize the ``create_database()`` function located
+in the ``Experiment`` class. For launching multiple databases, this function requires
+specifying a unique database identifier argument named ``db_identifier``. The ``db_identifier``
+argument plays a crucial role in SmartSim multi-database support by serving as a unique
+identifier for each database being launched. The ``db_identifier`` will be utilized later
+in the example.
+
```

Comment:
maybe reword for clarity and simplicity: "The function requires specifying a unique database identifier argument named ``db_identifier`` to launch multiple databases. The ``db_identifier`` serves as a unique identifier for each database being launched. Usage of the db_identifier is show in the example below. " Something like that?

Reply:
note to add where used in example